### DISCUSSION TITLE
Error when trying to upload config on Mac

### DISCUSSION THREAD

**Discussion by ifatfat** [kriomant/ch57x-keyboard-tool#106] [Q&A]
I've managed to compile the program but when I tried to upload the config to the keyboard, I have the following error: 
Error: claim interface

Caused by:
    Access denied (insufficient permissions)

I'm not sure if the mini keyboard is even connected, since it doesn't show up as a visible USB. 

I've tried disconnecting and connecting the mini keyboard but nothing happens. 

Would appreciate any help. Thanks!

**Comment by bluetardis** [comment]
did you try using 
`sudo ./ch57x-keyboard-tool etc etc etc
`
